docs/sdk.md

```diff
@@ -0,0 +1,8 @@
+---
+id: sdk
+title: SDK
+---
+
+this is another document
+
+need to link this to the proper place
```

Comment:
This page should display the contents from https://github.com/MainframeHQ/js-mainframe/blob/master/packages/sdk/README.md